README.md

```diff
@@ -20,7 +20,7 @@ comprised of the following components and features:
 
 # Usage
 
-    juju deploy cs:~containers/observable-kubernetes
+    conjure-up battlemidget/bundle-observable-kubernetes
```

Comment:
Not sure I agree with changing the default usage to `conjure-up` away from good old `juju deploy`. Could we add this as a supplemental way to deploy this bundle, along with a link or instructions on how to install conjure-up?